src/main/java/com/avail/optimizer/L2Optimizer.java

```diff
@@ -1200,9 +1592,49 @@ private void sanityCheck (final Interpreter interpreter)
 		 * computations.
 		 *
 		 * <p>Note that this breaks SSA by duplicating defining instructions.
-		 * </p>
+		 * It also always recomputes liveness after each change, so there's no
+		 * need to recompute it after this phase.</p>
```

Comment:
You need a `<p>` tag for this paragraph until we move to Kotlin.

Reply:
It's slightly dodgy, but I do intend both sentences to be part of the same paragraph.

Note:  This class has now been hoisted to a new file, OptimizationPhase.java.